writing-adapters.mdx

```diff
@@ -0,0 +1,101 @@
+---
+title: Adapter Writing Guide
+description: This guide provides a step-by-step approach to creating adapters for the Truf Network (TN). Adapters facilitate the ingestion of data from various sources into TN, leveraging the SDK and Prefect for efficient data management.
+---
+
+> **Note**: While this guide uses Python for examples, adapters can also be written in other languages like Go or JavaScript. Refer to the respective SDKs for more information:

```

Comment:
although the comment is true that we can insert data from other SDK, maybe the using the term

> adapters can also be written in another languages

could be misleading...

it's true people can build other frameworks in other languages, but maybe being more literal about is better? or am I off on some context?


Suggestion if we want to be more literal:
```suggestion
> **Note**: While adapters use a Python Framework, it's also possible to push data from other languages like Go or JavaScript. Refer to the respective SDKs for more information:
```

or of course any other way to make it more clear.

Reply:
Thank you for the suggestion!